proposals/2020/NMEC Implementation.md

```diff
@@ -0,0 +1,193 @@
+- The `auc:DerivedModel` will be a complex type that would look something like:
+```xml
+  <auc:DerivedModels>
+    <auc:DerivedModel ID="Model1">
+      <auc:DerivedModelPeriod>[Baseline, Reporting]</auc:DerivedModelPeriod>
+      <auc:DerivedModelBaselineID/>
+      <auc:DerivedModelInputs>
+        <auc:ModelType>[2P SLR, 3P heating CPM, 3P cooling CPM, 4P CPM, 5P CPM]</auc:ModelType>
+        <auc:NormalizationMethod>[Forecast, Backcast, Standard Conditions]</auc:NormalizationMethod>
+        <auc:BaselinePeriodStartDate>2016-12-01</auc:BaselinePeriodStartDate>
+        <auc:BaselinePeriodEndDate>2017-12-01</auc:BaselinePeriodEndDate>
+        <auc:ReportingPeriodStartDate/>
+        <auc:ReportingPeriodEndDate/>
+        <auc:IntervalFrequency>Monthly</auc:IntervalFrequency>
+        <auc:ResponseVariable>
+          <auc:EnergyResource>Electricity</auc:EnergyResource>
+          <auc:EnergyResourceUnits>kWh</auc:EnergyResourceUnits>
+          <auc:EndUse>All end uses</auc:EndUse>
+        </auc:ResponseVariable>
+        <auc:ExplanatoryVariables>
+          <auc:ExplanatoryVariable>
+            <auc:ExplanatoryVariableName>[Temperature, Relative Humidity, GHI, DHI, DNI, Time-of-week, Time-of-day,...]</auc:ExplanatoryVariableName>
+            <auc:ExplanatoryVariableUnits>[Fahrenheit, Percent, W/m2, Hour of week, Hour of day, ...]</auc:ExplanatoryVariableUnits>
+            <auc:StandardConditionDataSource>[TMY3, ...]</auc:StandardConditionDataSource>
+          </auc:ExplanatoryVariable>
+        </auc:ExplanatoryVariables>
+      </auc:DerivedModelInputs>
+      <auc:DerivedModelParameters>
+        <auc:BetaCooling>1.11</auc:BetaCooling>
+        <auc:BetaHeating>2.51</auc:BetaHeating>
+        <auc:BalancePointCooling>68</auc:BalancePointCooling>
+        <auc:BalancePointHeating>53</auc:BalancePointHeating>
+        <auc:Intercept>13.7</auc:Intercept>
+      </auc:DerivedModelParameters>
+      <auc:DerivedModelPerformance>
+        <auc:NDBE/>
+        <auc:RSquaredAdjusted/>
+        <auc:RSquared/>
+        <auc:RMSE/>
+        <auc:CVRMSE/>
+        <auc:MBE/>
+        <auc:NMBE/>
+      </auc:DerivedModelPerformance>
+      <auc:DerivedModelOutputs>
+        <auc:BaselinePeriodActualEnergyUse/>
+        <auc:ReportingPeriodActualEnergyUse/>
+        <auc:ReportingPeriodCalculatedEnergyUse/>
+        <auc:BaselinePeriodCalculatedEnergyUseStandardConditions/>
+        <auc:ReportingPeriodCalculatedEnergyUseStandardConditions/>
+        <auc:AvoidedEnergyUse/>
+        <auc:NormalizedEnergySavings/>
+        <auc:NumberOfParameters/>
+        <auc:NumberOfBaselinePeriodDataPoints/>
+        <auc:NumberOfReportingPeriodDataPoints/>
+        <auc:DegreesOfFreedom/>
+        <auc:UncertaintyOutputs>
+          <auc:SavingsPercentage/>
+          <auc:SavingsUncertainty/>
+        </auc:UncertaintyOutputs>
+        <auc:CalTrackOutputs>
+          <auc:Status>[Qualified, Unqualified, ...]</auc:Status>
+          <auc:Warnings>
```

Comment:
i like this

Reply:
Being specific to CalTrack was something stakeholders thought might be confusing to address initially, I concur.  Will consider down the line.  This was removed.